README.md

```diff
@@ -54,7 +54,7 @@ And then write a file inside your app root under `data/home/default.json`:
 Use the content inside an existing view, e.g. `app/views/spree/home/index.html.erb`:
 
 ```erb
-<% data = SolidusContent::Entry.data_for(:home, :default) %>
```

Comment:
I'll open an issue for implementing this, it's certainly an API I'd like to have in this extension